### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+
```
Comment (kasperl) [inline CODE_OF_CONDUCT.md]: Maybe drop the first empty newline?

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Feel free to ping us on more interactive support mediums to bring attention to
+your question.
+
+## Forum
+
+We use Github's [discussions](https://github.com/toitlang/toit/discussions) as
+our forum.
+
+## Real-time Chat
```
Comment (kasperl) [inline SUPPORT.md]: Chat -> chat

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Support for the Toit framework
+
+Welcome to Toit!
+
+# Documentation
+
+The [Toit docs](https://docs.toit.io) have lots of information. Some of it is
+not applicable to the open-source variant, but a lot of sections are variant-neutral.
+
+For example the [Language section](https://docs.toit.io/language) applies to the
```
Comment (kasperl) [inline SUPPORT.md]: Language -> language

File: SUPPORT.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# Support for the Toit framework
+
+Welcome to Toit!
+
+# Documentation
+
+The [Toit docs](https://docs.toit.io) have lots of information. Some of it is
```
Comment (kasperl) [inline SUPPORT.md]: I think the distinction between 'Toit docs' and toitdocs here is confusing. Maybe change this one to 'Toit documentation'?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,5 @@
-# .github
-Default community health files
+# Default community health files
+
+These files are used by default for subprojects that don't provide their own.
```
Comment (kasperl) [inline README.md]: Perhaps make this more Toit-centric. Change title to 'Default community health files for Toit' and 'used by default by projects in the [toitlang](https://github.com/toitlang) organization'.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,5 @@
...
+
+See [Github's community health files description](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file).
```
Comment (kasperl) [inline README.md]: GitHub's description of community health files

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (floitsch) [reply CODE_OF_CONDUCT.md]: done.

File: README.md (hunk 5 above)
Comment (floitsch) [reply README.md]: done.

File: README.md (hunk 6 above)
Comment (floitsch) [reply README.md]: done.

File: SUPPORT.md (hunk 4 above)
Comment (floitsch) [reply SUPPORT.md]: done.

File: SUPPORT.md (hunk 3 above)
Comment (floitsch) [reply SUPPORT.md]: done.

File: SUPPORT.md (hunk 2 above)
Comment (floitsch) [reply SUPPORT.md]: done.

**Pull request merged**